lighthouse-viewer/app/index.html

```diff
@@ -35,6 +35,11 @@ <h1 class="viewer-placeholder__heading">Lighthouse Report Viewer</h1>
       or <a href="https://web.dev/measure">web.dev/measure</a>.</div>
       <input type="url" class="viewer-placeholder__url js-gist-url"
              placeholder="Enter or paste Gist URL">
+
+      <br>
+      <b>DevTools users:</b> there is a <a href="https://github.com/GoogleChrome/lighthouse/issues/12841">known bug</a>
```

Comment:
No interest in the "no query param, no referrer" to narrow this to more likely devtools users? it's a lot of (somewhat embarrassing) text to show to the *definitely* uninterested majority

Reply:
> definitely uninterested majority

I disagree that this wouldn't be useful to the majority of traffic to this page.

You can ignore the traffic coming via the extension, because the entire page is blurred while loading. Same for gist/json. So you're left with people who go directly to the viewer, which will be a small fraction compared to the traffic that will come from DevTools soon (although, this remains to be seen).

Reply:
> You can ignore the traffic coming via the extension, because the entire page is blurred while loading. Same for gist/json. 

This is the traffic for whom I'm saying we should hide this message :) 

When something fails with the fetch for the 99.9% of users that *definitely* did not come from DevTools, seeing a message about DevTools that occupies over half the explanatory text feels unnecessary, that's all.